doc/api/fs.markdown

```diff
@@ -436,6 +436,15 @@ to the completion callback.
 
 Synchronous fchown(2). Returns `undefined`.
 
+## fs.fdatasync(fd, callback)
+
+Asynchronous fdatasync(2). No arguments other than a possible exception are given
```

Comment:
81, you're right :)
Will fix. There are more lines in this file beyond 80 chars btw, but that's for another PR.